### PR TITLE
add iso-ara-arab-latn-233-1984

### PR DESCRIPTION
**Pull request by AhMohsen46**

### CODE REVIEW

**Comment by ronaldtse** [comment]
Thanks @AhMohsen46 !

File: maps/iso-ara-arab-latn-233-1984.yaml [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+authority_id: ISO
+id: 1984
```
Comment (ronaldtse) [inline maps/iso-ara-arab-latn-233-1984.yaml]: `id` should be `233-1984`.

File: maps/iso-ara-arab-latn-233-1984.yaml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+language: ara
+source_script: Arab
+destination_script: Latn
+name: ROMANIZATION OF ARABIC -- ISO 1984 System
+url:
+  - http://transliteration.eki.ee/pdf/Arabic_2.2.pdf
```
Comment (ronaldtse) [inline maps/iso-ara-arab-latn-233-1984.yaml:10]: Add first reference: `https://www.iso.org/standard/4117.html`

File: maps/iso-ara-arab-latn-233-1984.yaml [hunk 3]
```diff
@@ -0,0 +1,171 @@
+---
+authority_id: ISO
+id: 1984
+language: ara
+source_script: Arab
+destination_script: Latn
+name: ROMANIZATION OF ARABIC -- ISO 1984 System
```
Comment (ronaldtse) [inline maps/iso-ara-arab-latn-233-1984.yaml]: name should be: `ISO 233:1984 Documentation — Transliteration of Arabic characters into Latin characters`

File: maps/iso-ara-arab-latn-233-1984.yaml [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+url:
+  - http://transliteration.eki.ee/pdf/Arabic_2.2.pdf
+  - http://www.eki.ee/wgrs/rom1_ar.pdf
+creation_date: 1984
+confirmation date: 2018-06
+description: |
+  no description provided
```
Comment (ronaldtse) [inline maps/iso-ara-arab-latn-233-1984.yaml]: Description from the ISO page:

"Is one of a series of International Standards dealing with the conversion of systems of writing, following the principles of stringent conversion in order to permit international information exchange. Its aim is to provide a means for international communication of written messages in a form which permits the automatic transmission and reconstitution of these by men or machines. Cancels and replaces ISO Recommendation R 233-1961."

File: maps/iso-ara-arab-latn-233-1984.yaml [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+    expected: ǧumhuwriyaẗ miṣr al ‘arabiyaẗ
+
+  - source: الرِيَاض
+    expected: ar riya’ḍ
+
+  - source: الشارِقة
+    expected: aš šâriqaẗ
+
+map:
+  inherit: "un-ara-Arab-Latn-2017"
```
Comment (ronaldtse) [inline maps/iso-ara-arab-latn-233-1984.yaml]: Can we implement this map as an individual map (i.e. not inheriting from UN)? Let's only use inheritance in the same series (e.g. UN 1971 -> UN 1972). Thanks.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: maps/iso-ara-arab-latn-233-1984.yaml (hunk 5 above)
Comment (AhMohsen46) [reply maps/iso-ara-arab-latn-233-1984.yaml]: yes sure, this makes more sense, thanks!

Comment (ronaldtse) [reply maps/iso-ara-arab-latn-233-1984.yaml]: OK let me know when this is done. Thanks!

File: maps/iso-ara-arab-latn-233-1984.yaml [hunk 6]
```diff
@@ -0,0 +1,179 @@
+---
+authority_id: iso
+id: 233-1984
+language: ara
+source_script: Arab
+destination_script: Latn
+name: ISO 233:1984 Documentation — Transliteration of Arabic characters into Latin characters
+url:
+  - https://www.iso.org/standard/4117.html
```
Comment (AhMohsen46) [inline maps/iso-ara-arab-latn-233-1984.yaml:9]: @ronaldtse I guess it's done, but it didn't reflect in the comment as the line you commented on was not changed

**Comment by ronaldtse** [comment]
Thanks @AhMohsen46 !

**Pull request merged**